index.html

```diff
@@ -2458,14 +2458,24 @@ <h3>Data Schemas</h3>
 
         <dl>
           <dt><var>credentialSchema</var></dt>
-          <dd>
-The value of the <code>credentialSchema</code> <a>property</a> MUST be one or
-more data schemas that provide <a>verifiers</a> with enough information to
-determine if the provided data conforms to the provided schema. Each
-<code>credentialSchema</code> MUST specify its <code>type</code> (for example,
-<code>JsonSchemaValidator2018</code>), and an <code>id</code> <a>property</a>
-that MUST be a <a>URL</a> identifying the schema file. The precise contents of
-each data schema is determined by the specific type definition.
+          <dd><p>
+The value of the <code>credentialSchema</code> <a>property</a> MUST be a single
+data schema object which provides <a>verifiers</a> with enough information 
+to determine if the provided data conforms to the provided schema object. Each
+<code>credentialSchema</code> MUST specify its <code>type</code> (e.g.,
```

Comment:
Past Internationalization reviewers have asked us to not use "e.g.," and use "for example" instead. "e.g.," is difficult for some non-native English readers (or readers that are not familiar with Latin). I don't remember if we've done a scan of the spec to remove all examples of "e.g.," with "for example".

Reply:
thanks, I saw there's a mix of e.g. and for example. I'll change this one.

Reply:
Something similar should be done with instances of `i.e.`. One possible replacement is `in other words`, either parenthetically or (typically) preceded by a semicolon and followed by a comma.

Reply:
Thanks both, created https://github.com/w3c/vc-data-model/issues/1025 to track

Reply:
I am having difficult determining where we have  got to with this clarification. First, I had no problem understanding the original language. Second, we already have examples e.g. JFF plugfest, or using different schema languages (i.e.types) to define the same VC data structure. So if type does equal the schema language, then each type should have a single ID (URL) where the schema can be downloaded from. So what exactly is the change that is being proposed now? Since I cannot see how one URL/ID can or should point to different schema files.

Reply:
@David-Chadwick see this for an example https://github.com/w3c-ccg/vc-json-schemas/issues/121

Reply:
The I18N WG is not aware of our having--now or in the past--a policy against (mostly Latin derived) abbreviated terms common to English such as `i.e.`, `e.g.`, `cf.`, `etc.` and so forth. If one or another of us made this comment in the past, this is to say: we didn't mean it/sorry.

I do note that some writers or organizations have style guides that prefer spelled out English terms (for clarity in English), but I18N does not take a position on this. I double-checked the [W3C Manual of Style](https://www.w3.org/Guide/manual-of-style/) and it is silent on this topic, so I suppose it is not a W3C policy either.

Reply:
*cries* it came up during the VC 1.0 review and I had to go through the entire spec to make the conversion, IIRC. I'd have to go and track it down... which, as you state above, would probably not change the guidance at this point.

So, now this is a VCWG / Editor's decision -- how do we want to be consistent here... always use abbreviated latin terms "e.g.," and "i.e.," or always use the spelled out English equivalents?

Reply:
I think it's simpler to use latin. There will be less variation since we'll need to pick common english replacements which don't necessarily come to mind without looking at examples.

Reply:
resolved by the discussion in #1045